src/include/duckdb/execution/physical_operator.hpp

```diff
@@ -85,7 +86,9 @@ class PhysicalOperator {
 public:
 	PhysicalOperator(PhysicalOperatorType type, vector<LogicalType> types, idx_t estimated_cardinality)
 	    : type(type), types(std::move(types)), estimated_cardinality(estimated_cardinality) {
+		ph_join_stats = make_unique<JoinStats>();
```

Comment:
Does not make much sense that every physical operator gets a `JoinStats` field. It only has estimated cost/cardinality. Can we rename this to something more general, i.e., `EstimatedProperties`?

Reply:
The same goes for the logical operators